addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -1035,6 +1046,21 @@ def _recompute_move_entries_totals(env):
     )
 
 
+def fill_account_move_line_missing_fields(env):
```

Comment:
This is not needed. Odoo already performs the exact same query since https://github.com/odoo/odoo/pull/65232

Reply:
It is needed, because this fields are computed https://github.com/odoo/odoo/blob/13.0/odoo/fields.py#L864

Reply:
They are related AFAIK: https://github.com/odoo/odoo/blob/917be4131491dada93f8cb9cd9cc1ae4ce731f2b/addons/account/models/account_move.py#L2644

Reply:
They are related but the related field cannot be computed

Reply:
OK, then no problem to put this that way.